src/WebApiToTypeScript/Endpoints/EndpointsService.cs

```diff
@@ -111,11 +111,13 @@ private void WriteInterfaceWithCallToBlock(TypeScriptBlock interfaceWithCallBloc
         {
             var callArguments = action.BodyParameters;
 
-            var callArgumentStrings = callArguments
+            var callArgument = callArguments
                 .Select(a => a.GetParameterString(withOptionals: false, interfaceName: true))
                 .SingleOrDefault();
 
-            var callArgumentsList = string.Join(", ", callArgumentStrings);
+            var callArgumentsList = string.IsNullOrWhiteSpace(callArgument)
+                ? "httpConfig?"
```

Comment:
We should add `: angular.IRequestShortcutConfig` in these two places as well since that is the main interface point for API users. right?